code/game/gamemodes/miniantags/abduction/abductee_objectives.dm

```diff
@@ -148,3 +148,68 @@
 
 /datum/objective/abductee/sixthsense
 	explanation_text = "You died back there and went to heaven... or is it hell? No one here seems to know they're dead. Convince them, and maybe you can escape this limbo."
+
+/datum/objective/abductee/forbiddennumber
+	explanation_text = "Ignore anything in a set number of"
+
+/datum/objective/abductee/forbiddennumber/New()
+	..()
+	var/number = pick(list("two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"))
+	explanation_text +=" [number], they don't exist."
+
+/datum/objective/abductee/buddy
+	explanation_text = "Being alone and in large groups are both frightening. Try to be alone with only one other person whenever possible."
```

Comment:
```suggestion
	explanation_text = "Being alone and in large groups is both frightening. Try to be alone with only one other person whenever possible."
```
I think this has to be singular?

Reply:
Are is correct in this instance I think

Reply:
Yeah there was a discussion about this in #coding_chat, I'm blaming dutch grammar rules in which my review is correct :D